test/system/ts.mjs

```diff
@@ -1,14 +1,16 @@
 import tap from 'tap';
 import * as child from 'child_process';
 
+import {promises as fs} from 'fs';
+
 import {cleanSnapshot} from '../test-utils.mjs';
 
 import {promisify} from 'util';
 
 cleanSnapshot();
 const execFile = promisify(child.execFile);
 
-tap.test('TS System Tests', async t => {
+tap.test('TS System Tests', {saveFixture: true}, async t => {
```

Comment:
Should the fixture dir be cleaned up if it's created by the test?

Reply:
Correct, was left behind from some prototyping I did that is now on another branch. Removeed.